misk/src/main/kotlin/misk/metrics/Metrics.kt

```diff
@@ -38,14 +38,12 @@ class Metrics @Inject internal constructor(val registry: CollectorRegistry) {
   }
 
   fun histogram(
-    name: String,
-    help: String = "",
-    labelNames: List<String> = listOf(),
-    buckets: DoubleArray? = null
-  ): Histogram {
-    val builder = Histogram.build(name, help).labelNames(*labelNames.toTypedArray())
-    if (buckets != null) builder.buckets(*buckets)
-    return builder.register(registry)
+          name: String,
+          help: String = "",
+          labelNames: List<String> = listOf(),
+          buckets: DoubleArray? = null
+  ): misk.digester.Histogram {
```

Comment:
you shouldn't need the fully qualified `Histogram` name here. just `Histogram` should be fine right?

Reply:
There was arbitration from import io.prometheus.client.*, I'll change the imports to exclude prometheus.Histogram